### ISSUE TITLE
Cuda 11

### ISSUE THREAD

**Issue by quentingllmt** [divamgupta/image-segmentation-keras#260]
Hi, I'm trying to run this repository on an NVIDIA Ampere GPU (RTX 3090).

This GPU needs Cuda 11.x which needs tensorflow 2.4. 

Is the project compatible as keras is now part of tensorflow ? As anyone already done this ?

**Comment by Marius-Juston** [comment]
Yes this works with CUDA 11.x with tensorflow 2.4. I have done it

**Comment by sotomotocross** [comment]
> 

So you installed CUDA 11.x with tensorflow-gpu==2.4.1?
What are the versions of the rest packages in order to run properly the image-segmnentation-keras?
I am trying with an RTX 30xx, CUDA 11.x and tensorflow-gpu==2.4.1 but it doesn't recognize my GPU.
